angular/src/lib/editable-textfield/editable-textfield.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { InputModule } from '../input';
+import { EditableTextfieldComponent } from './editable-textfield.component';
+
+@NgModule({
+  declarations: [EditableTextfieldComponent],
+  imports: [
+    CommonModule,
+    InputModule,
+    FormsModule,
+    ReactiveFormsModule
+  ],
+  exports: [EditableTextfieldComponent]
```

Comment:
I think we are trying to add 
`schemas: [ CUSTOM_ELEMENTS_SCHEMA ]`
to all component modules and test modules

Reply:
Added, thanks